src/olympia/lib/settings_base.py

```diff
@@ -1121,9 +1144,6 @@ def JINJA_CONFIG():
         'queue': 'addons'},
     'olympia.addons.tasks.version_changed': {'queue': 'addons'},
 
-    # Files (goes to devhub queue).
-    'olympia.files.tasks.update_webext_descriptions_all': {'queue': 'devhub'},
```

Comment:
Can we have that change in a separate commit so that we can easily rebase it later after testing?